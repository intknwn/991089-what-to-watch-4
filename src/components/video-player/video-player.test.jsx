import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';

it(`VideoPlayer component should renders correctly`, () => {
  const tree = renderer.create(
      <VideoPlayer
        src={`https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`}
        width={280}
        height={175}
        poster={`https://via.placeholder.com/273x410`}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
