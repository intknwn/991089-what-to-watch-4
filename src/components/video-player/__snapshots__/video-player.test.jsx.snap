// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer component should renders correctly 1`] = `
<video
  autoPlay={true}
  height={175}
  loop={true}
  muted={true}
  poster="https://via.placeholder.com/273x410"
  src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
  width={280}
/>
`;
